testing/src/main/java/org/motechproject/nms/testing/web/TestingController.java

```diff
@@ -62,15 +62,21 @@ public String createSubscriptionPacks() {
 
     @RequestMapping(value = "/createMctsMoms")
     @ResponseBody
-    public String createMctsMoms(@RequestParam(required = true) int count) throws IOException {
-        return testingService.createMctsMoms(count);
+    public String createMctsMoms(
+            @RequestParam(required = true) int count,
```

Comment:
kosh == :cold_sweat: right now, but after reading [this](http://stackoverflow.com/questions/394809/does-python-have-a-ternary-conditional-operator), kosh == :sunglasses: